src/Umbraco.Core/Extensions/ConnectionStringExtensions.cs

```diff
@@ -1,35 +1,24 @@
 // Copyright (c) Umbraco.
 // See LICENSE for more details.
 
-using System;
-using Microsoft.Extensions.Configuration;
-using Umbraco.Cms.Core;
 using Umbraco.Cms.Core.Configuration.Models;
 
-namespace Umbraco.Extensions
-{
-    public static class ConnectionStringExtensions
-    {
-        public static bool IsConnectionStringConfigured(this ConnectionStrings connectionString)
-            => connectionString != null &&
-            !string.IsNullOrWhiteSpace(connectionString.ConnectionString) &&
-            !string.IsNullOrWhiteSpace(connectionString.ProviderName);
-
-        /// <summary>
-        /// Gets a connection string from configuration with placeholders replaced.
-        /// </summary>
-        public static string? GetUmbracoConnectionString(
```

Comment:
This extension method moved to `ConfigurationExtensions` that's in the same namespace.

Reply:
Still needs to be documented on the PR description